bronze.py

```diff
@@ -0,0 +1,45 @@
+from flask_dance.consumer import OAuth2ConsumerBlueprint
+from functools import partial
+from flask.globals import LocalProxy, _lookup_app_object
+
+try:
+    from flask import _app_ctx_stack as stack
+except ImportError:
+    from flask import _request_ctx_stack as stack
+
+def make_bronze_blueprint(
+    scope,
+    redirect_url=None,
+    redirect_to=None,
+    login_url=None,
+    authorized_url=None,
+    session_class=None,
+    storage=None,
+):
+    bronze_bp = OAuth2ConsumerBlueprint(
+        "bronze",
+        __name__,
+        client_id='TestClient',
```

Comment:
these will ultimately need to come from environment vars

Reply:
changed it.